lib/ui/painting/multi_frame_codec.cc

```diff
@@ -115,17 +115,17 @@ sk_sp<DlImage> MultiFrameCodec::State::GetNextFrameImage(
           << "Frame " << nextFrameIndex_ << " depends on frame "
           << requiredFrameIndex
           << " and no required frames are cached. Using blank slate instead.";
+      return nullptr;
```

Comment:
It would be more graceful to just draw the frame against the zeroed bitmap in this case instead of failing the frame.

Reply:
Done